manifests/charts/global.yaml

```diff
@@ -51,7 +51,6 @@ global:
 
   # Enabled by default in master for maximising testing.
   istiod:
-    enabled: true
     enableAnalysis: false
```

Comment:
what does enableAnalysis do?

Reply:
It's about putting istioctl analyze message into the status of CRs
. it's not changes in this pr

Reply:
thanks!